services/site/src/pages/about.tsx

```diff
@@ -0,0 +1,194 @@
+import Footer from "app/components/Footer";
+import A11yphantLogoWithoutText from "app/components/icons/A11yphantLogoWithoutText";
+import Navigation from "app/components/Navigation";
+import { usePrefersReducedMotion } from "app/hooks/prefersReducedMotion";
+import clsx from "clsx";
+import Head from "next/head";
+import Image from "next/image";
+import React from "react";
+
+import a11yphantAnimation from "../../public/images/a11yphant-animation.gif";
+import a11yphantStatic from "../../public/images/a11yphant-static.jpg";
+
+const About: React.FunctionComponent = () => {
+  const prefersReducedMotion = usePrefersReducedMotion();
+
+  return (
+    <>
+      <Head>
+        <title>About | a11yphant</title>
+        <meta name="robots" content="noindex,nofollow" />
```

Comment:
Remove the meta tag for robots as we want this page indexed and listed in search results.